frontend/src/lib/apiClient.js

```diff
@@ -75,6 +75,18 @@ const getRecipes = (search) => {
 
 const getRecipe = id => apiClient.get(`api/v1/recipes/${id}`);
 
+const createRecipe = recipe => apiClient.post('api/v1/recipes#create', {
+  recipe: {
+    title: recipe.title,
+    snippet: recipe.description,
+    difficulty: recipe.difficulty,
+    duration: recipe.duration,
+    servings: recipe.servings,
+    user_id: recipe.currentUserId,
```

Comment:
As mentioned in the backend code, you'll want to remove this. You will need to send the authorization header, though.